linguatec_lexicon/views.py

```diff
@@ -80,6 +80,25 @@ class DefaultLimitOffsetPagination(LimitOffsetPagination):
     max_limit = 100
 
 
+class LexiconViewSet(viewsets.ReadOnlyModelViewSet):
+    """
+    API endpoint that allows lexicons to be viewed.
+    """
+    queryset = Lexicon.objects.all().order_by('name')
+    serializer_class = LexiconSerializer
+    pagination_class = DefaultLimitOffsetPagination
+
+    @action(detail=False)
+    def get_lexicon_names(self, request):
+        queryset = Lexicon.objects.all().order_by('name')
+        page = self.paginate_queryset(queryset)
+        if page is not None:
+            serializer = self.get_serializer(page, many=True)
+            return self.get_paginated_response(serializer.data)
+
+        serializer = self.get_serializer(queryset, many=True)
+        return Response(serializer.data)
+
```

Comment:
I think that this method duplicates the response provided by default `list()` method.
If you need to customize queryset order_by you could override `get_queryset()` method.

Reply:
Yeah this method was like an overwritting of the default method, i have deleted this and refactored the default method